addons/sys_zeus/ACE_Actions.hpp

```diff
@@ -0,0 +1,17 @@
+class ACE_ZeusActions {
```

Comment:
Might be cleaner if we add all actions via SQF under "if ACE running...", would also lower size and amount of files.

Look at how racks and intercom do it.